src/main/java/net/dv8tion/jda/api/entities/Message.java

```diff
@@ -2657,6 +2666,26 @@ public boolean isEphemeral()
             return ephemeral;
         }
 
+        /**
+         * Gets the waveform data encoded in this message. This is currently only present on
+         * {@link MessageFlag#IS_VOICE_MESSAGE voice messages}.
+         *
+         * @return A possibly-{@code null} unmodifiable list of integers representing the volume
+         *         of the audio at the given point. Volume is sampled at a rate of 10 times per
```

Comment:
Probably best to just say `8bit 10Hz Mono PCM` and maybe provide an `AudioFormat` constant for it.